bin/tests/common.rs

```diff
@@ -172,7 +184,7 @@ pub fn create_dirs<P: AsRef<Path>>(dirs: &[P]) {
 
 pub fn open_files_include(id: u32, file: &PathBuf) -> Option<String> {
     let child = Command::new("lsof")
-        .args(&["-p", &id.to_string()])
+        .args(&["-l", "-p", &id.to_string()])
```

Comment:
lsof was showing in the output "no pwd entry for UID" which was irritating, I've fixed it here even though it's unrelated to this PR.